scenario/scenario_instance.json

```diff
@@ -0,0 +1,275 @@
+      "@value": "0",
+      "@type": "xsd:unsignedInt"
+    },
+    "scenario:permanentTrafficObjects": {
+      "@value": "1",
+      "@type": "xsd:unsignedInt"
+    },
+    "scenario:movementDescription": {
+      "@value": "FollowTrajectory",
+      "@type": "xsd:string"
+    },
+    "scenario:controllers": {
+      "@value": "SampleControler1",
+      "@type": "xsd:string"
+    },
+    "scenario:customCommands": {
+      "@value": "UserDefinedAction: TurnAround",
+      "@type": "xsd:string"
+    }
+  },
+  "scenario:environmental": {
+    "@type": "scenario:Environmental",
+    "scenario:sunAzimuth": {
+      "@value": "3.1415",
+      "@type": "xsd:float"
+    }
+  },
+  "scenario:traffic": {
+    "@type": "scenario:Traffic",
+    "scenario:country": {
+      "@value": "Germany",
+      "@type": "xsd:string"
+    },
+    "scenario:countrySpecificTP": {
+      "@value": "CustomCar",
+      "@type": "xsd:string"
+    },
+    "scenario:countrySpecificSign": {
+      "@value": "Aircraft",
+      "@type": "xsd:string"
+    }
+  },
+  "scenario:structural": {
+    "@type": "scenario:Structural",
+    "scenario:catalogs": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_pedestrian_catalog.xosc",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    },
+    "": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_car_model.ogb",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    },
+    "scenario:trafficSpace": {
+      "@type": "general:Link",
+      "scenario:url": {
+        "@value": "https://inavlid.link.com/my_map.xodr",
+        "@type": "xsd:anyURI"
+      },
+      "scenario:type": "Document"
+    }
+  },
+  "scenario:format": {
+    "@type": "scenario:Format",
+    "scenario:dataFormat": "ASAM OpenSCENARIO",
+    "scenario:version": {
+      "@value": "1.2",
+      "@type": "xsd:string"
+    }
+  },
+  "general:general": {
+    "@type": "general:General",
+    "general:DataResource": {
+      "@type": "gx:DataResource",
+      "gx:expirationDateTime": {
+        "@value": "2100-01-01T00:00:00",
+        "@type": "xsd:dateTime"
+      },
+      "gx:containsPII": false,
+      "gx:exposedThrough": {
+        "@id": "https://inavlid.link.com/my_scenario_resource.zip"
+      },
+      "gx:producedBy": {
+        "@id": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base"
+      },
+      "gx:policy": {
+        "@value": "MPL-2.0",
+        "@type": "xsd:string"
+      },
+      "gx:name": {
+        "@value": "MyScenario",
+        "@type": "xsd:string"
+      },
+      "gx:description": {
+        "@value": "A sample description of a none existing scenario.",
+        "@type": "xsd:string"
+      }
+    },
+    "general:Links": {
+      "@type": "general:Links",
+      "general:data": {
+        "@type": "general:Link",
+        "scenario:url": {
```

Comment:
When type "general:Links" is used, also url and type needs to be from namespace general as they are defined there

Reply:
I only mark it here, but it appears a few times

Reply:
@heuerfin can you change this?

Reply:
Fixed in b618f4e